Library/Formula/bork.rb

```diff
@@ -0,0 +1,14 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.tar.gz"
+  sha256 "3e8217d5fe4de94663dac74af505d05649c9d995998d19d4b247e5d3ecb1dd4b"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match /bork usage:/, shell_output("#{bin}/bork", 1)
```

Comment:
`testpath` is defined in the `test` block; you don’t need to pass it yourself :wink: 
See e.g. [`cookiecutter.rb`](https://github.com/Homebrew/homebrew/blob/80d1a2c7bd45a5979d0fb914178b9bbe3eb7d1ee/Library/Formula/cookiecutter.rb#L88) but there are a lot of formulae that use this variable.